site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1348,7 +1348,7 @@ public function renderDiscussionForum($threadIds, $submitter, $isTeam = false) {
             }
 
             $posts_view .= <<<HTML
-                    <a href="{$this->core->buildCourseUrl(['forum', 'threads', $threadId])}" target="_blank" rel="noopener nofollow" class="btn btn-default btn-sm" style="margin-top:15px; text-decoration: none;" onClick=""> Go to thread</a>
```

Comment:
does removing the target change the functionality in any way?

Reply:
Like when we have target it will open the thread page in some other tab
and without target it will open the page in the same tab (target="_blank")

Reply:
This PR is supposed to only be adding testing, not changing the functionality

Reply:
Could  you re-add the target to restore original functionality?

Reply:
done